_tests/managed_tests/tests/__snapshots__/Wooe2eTest__test_woo_e2e_no_op_php82_woorc_php82_wpstable_b23dfeb910e3a66fc64cf81023cfeaea__1.php

```diff
@@ -12446,6 +12447,42 @@
                     {
                         "count": "1",
                         "message": "PHP Deprecated: Function Automattic\\\\WooCommerce\\\\Admin\\\\WCAdminHelper::is_store_page is deprecated since version 9.8.0! Use is_current_page_store_page instead. in \\/var\\/www\\/html\\/wp-includes\\/functions.php on line 6114"
+                    },
```

Comment:
@Luc45 Seems legit?

Reply:
Maybe the Manager you tested against to generate these snapshots was outdated/didn't have the commit that parsed the `auto` value?

Reply:
Just make sure to deploy the latest commits to Staging and `php QITSelfTests.php update phpcompatibility`

Reply:
This was an actual bug, it's fixed on https://github.com/Automattic/compatibility-dashboard/pull/1289, it was caused by a merge conflict

Reply:
Fixed and snapshots updated, all good now.